package.json

```diff
@@ -32,6 +32,14 @@
     "test-multibrowser": "grunt test:multibrowser",
     "test-snyk": "snyk test"
   },
+  "files": [
+    "bin",
+    "build",
+    "lib",
+    "index.js",
```

Comment:
It might be then ok to remove `lib` since its contents is being build and available in `build` directory.